src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -123,4 +123,21 @@ public void doGet(
     }
     return events;
   }
+
+  @Override
```

Comment:
Some javadoc here describing what is expected in the request and what is sent back to the client would be nice.

Reply:
Agreed, `Resolved`